src/Microsoft.Data.Analysis.Interactive/Microsoft.Data.Analysis.Interactive.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
+		<IsPackable>false</IsPackable>
+		<NoWarn>$(NoWarn);MSML_ParameterLocalVarName;SA1028</NoWarn>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.0.0">
```

Comment:
No hardcoded versions in project files.  We should be defining centrally.

Reply:
All the version info is in this file. https://github.com/dotnet/machinelearning/blob/master/eng/Versions.props